crates/bonsaidb-core/src/connection.rs

```diff
@@ -982,3 +942,33 @@ pub struct Database {
     /// The schema defining the database.
     pub schema: SchemaName,
 }
+
+/// A plain-text password. This struct automatically overwrites the password
+/// with zeroes when dropped.
```

Comment:
I can't see where it does. You have to use `ZeroizeOnDrop` for that. You can skip using `Zeroize`.

Reply:
Oops. I'll fix with the old way of zeroizing. Sadly, trying to use 1.5 introduces dependency conflicts, and I'm not really wanting to bring in a new version just for this one use. Since it's not an invariant-unsafe zeroizable type, I figured it's fine using the old implementation.

But in the process of figuring that out I forgot to add the extra bit to actually make it happen on drop. Great catch.